app/views/template/Layout.scala.html

```diff
@@ -61,8 +61,8 @@
 }
 
 @scripts = {
-<script src='@controllers.routes.Assets.versioned("lib/govuk-frontend/govuk/all.js")'></script>
-<script src='@controllers.routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
+<script @CSPNonce.attr src='@controllers.routes.Assets.versioned("lib/govuk-frontend/govuk/all.js")'></script>
+<script @CSPNonce.attr src='@controllers.routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
```

Comment:
The `@CSPNonce.attr` needs to also be added to the script below on line 66 otherwise it appears to cause failures with the loading of the CSP. You can see this by viewing the console when running the service